hedera-node/src/test/java/com/hedera/services/txns/crypto/CryptoUpdateTransitionLogicTest.java

```diff
@@ -422,16 +470,39 @@ void acceptsAliasedAccountIDToUpdate() {
 				.build();
 		given(ledger.exists(PAYER)).willReturn(true);
 		given(ledger.lookupAliasedId(aliasAccountPayer, INVALID_ACCOUNT_ID)).willReturn(AliasLookup.of(PAYER, OK));
-		given(ledger.lookupAndValidateAliasedId(aliasAccountPayer, INVALID_ACCOUNT_ID)).willReturn(AliasLookup.of(PAYER, OK));
+		given(ledger.lookupAndValidateAliasedId(aliasAccountPayer, INVALID_ACCOUNT_ID)).willReturn(
+				AliasLookup.of(PAYER, OK));
 
 		given(accessor.getTxn()).willReturn(cryptoUpdateTxn);
 		given(txnCtx.accessor()).willReturn(accessor);
+		given(ledger.alias(PAYER)).willReturn(aliasAccountPayer.getAlias());
 
 		subject.doStateTransition();
 
 		verify(txnCtx).setStatus(SUCCESS);
 	}
 
+	@Test
+	void failsTryingToUpdateAlias() {
```

Comment:
how is this different from 
```@Test
 	void rejectsUpdateIfAliasAlreadyPresent()

Reply:
Agreed @anighanta . Removed the duplicated test.